app/ui-react/packages/ui/src/Data/Virtualizations/Views/ViewCreateLayout.tsx

```diff
@@ -69,59 +69,58 @@ export const ViewCreateLayout: React.FunctionComponent<
   isLastStep = false,
 }: IViewCreateLayoutProps) => {
   return (
-    <div
-      className={classnames(
-        'wizard-pf-body view-create-layout syn-scrollable',
-        {
-          'has-footer': true,
-        }
-      )}
-    >
-      {header}
-      <div className="wizard-pf-row view-create-layout__body syn-scrollable--body">
-        <div
-          className={
-            'wizard-pf-main cards-pf view-create-layout__contentWrapper'
-          }
-        >
-          <div className="view-create-layout__content">{content}</div>
+    <div className={'integration-editor-layout'}>
+      <div className={'integration-editor-layout__header'}>{header}</div>
+      <div className={'integration-editor-layout__body'}>
+        <div className={'integration-editor-layout__contentOuter'}>
+          <div className={'integration-editor-layout__contentInner'}>
+            <PageSection>
+                <Card>
+                  <CardBody className="view-create-layout-card-body">
+                    {content}
+                  </CardBody>
+                  <CardFooter className="view-create-layout-card-footer">
+                  <ButtonLink
+                      data-testid={'view-create-layout-back-button'}
+                      onClick={onBack}
+                      href={backHref}
+                      className={'wizard-pf-back'}
+                    >
+                      <i className="fa fa-angle-left" /> Back
+                    </ButtonLink>
+                    &nbsp;
+                    <ButtonLink
+                      data-testid={'view-create-layout-next-button'}
+                      onClick={onNext}
+                      href={nextHref}
+                      as={'primary'}
+                      className={'wizard-pf-next'}
+                      disabled={isNextLoading || isNextDisabled}
+                    >
+                      {isNextLoading ? <Loader size={'xs'} inline={true} /> : null}
+                      {isLastStep ? (
+                        'Done'
```

Comment:
This file and `ViewsImportLayout` should be i18n'd. Maybe open up another issue for that.